### PR TITLE
Adding IN operator feature doc

### PR DESCRIPTION
**Pull request by AlanWong-MS**
### Description
Adding IN operator feature doc

### CODE REVIEW

File: _posts/2018-07-03-04-26-InOperator.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title : "4.26 IN Operator"
+layout: post
+category: "4. OData features"
+---
+
+# IN Operator
+
+Starting in WebAPI OData V7.0.0 [[ASP.NET](https://www.nuget.org/packages/Microsoft.AspNet.OData/7.0.0) | [ASP.NET Core](https://www.nuget.org/packages/Microsoft.AspNetCore.OData/)], the IN operator is a [supported feature](http://docs.oasis-open.org/odata/new-in-odata/v4.01/cn01/new-in-odata-v4.01-cn01.html#_Toc485385090 "new feature") that enables a shorthand way of writing multiple EQ expressions joined by OR. For example,
```
Comment (biaol-odata) [inline _posts/2018-07-03-04-26-InOperator.md]: nit: is the "new feature" for hovering text needed?

File: _posts/2018-07-03-04-26-InOperator.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+`GET /service/Products?$filter=Name in ('Milk', 'Cheese', 'Donut')`
+
+Of the binary expression invoking IN, the left operand must be a single value and the right operand must be a comma-separated list of primitive values or a single expression that resolves to a collection; the expression returns true if the left operand is a member of the right operand.
+
+# Usage
+
+IN operator is supported only for $filter at the moment and hardcoded collections are supported with parentheses. See examples below.
+
+```
+~/Products?$filter=Name in ('Milk', 'Cheese')
```
Comment (biaol-odata) [inline _posts/2018-07-03-04-26-InOperator.md]: can we add an example for parameter alias for collection values?

Comment (biaol-odata) [reply _posts/2018-07-03-04-26-InOperator.md]: :shipit:

File: _posts/2018-07-03-04-26-InOperator.md (hunk 1 above)
Comment (AlanWong-MS) [reply _posts/2018-07-03-04-26-InOperator.md]: Thanks for the catch. It is not needed.

File: _posts/2018-07-03-04-26-InOperator.md (hunk 2 above)
Comment (AlanWong-MS) [reply _posts/2018-07-03-04-26-InOperator.md]: Parameter alias is actually not supported for collection values in ODL side. I opened an [issue](https://github.com/OData/odata.net/issues/1161) to track this.

**Pull request merged**